azure/eventhub_log_forwarder/parent_template.json

```diff
@@ -0,0 +1,101 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```

Comment:
We should have the functionapp and function name in here too right?

Reply:
Yep, I've added this now!